### PR TITLE
[PR-12263][SEO] Meta description for docs pages

### PR DESCRIPTION
**Pull request by AlexKom9**
## Demo

http://void-docs.talkable.com/

## Related Stories

[](https://talkable.atlassian.net/browse/PR-12263)

### CODE REVIEW

**Comment by AlexKom9** [comment]
@zhuravel review please

File: source/advanced_features/referrals_approval.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
 .. _advanced_features/referrals_approval:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: The approvals process help to automatically detect fake new user accounts. Cheaters won’t get through the system.
```
Comment (zhuravel) [inline source/advanced_features/referrals_approval.rst]:
```suggestion
   :description: The approvals process helps automatically detect fake new user accounts. Cheaters won’t get through the system.
```

File: source/advanced_features/subscribing_to_events.rst [hunk 2]
```diff
@@ -1,6 +1,9 @@
 .. _advanced_features/subscribing_to_events:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Talkable JS integration script embeds Talkable campaign as an iframe with src attribute starting with www.talkable.com domain.
```
Comment (zhuravel) [inline source/advanced_features/subscribing_to_events.rst]: This is not true for white-labeled integrations.
```suggestion
   :description: Talkable JS integration script embeds Talkable campaign as an iframe.
```

File: source/android_sdk.rst [hunk 3]
```diff
@@ -1,6 +1,9 @@
 .. _android_sdk:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Talkable has an Android framework enabling you to use Talkable in Andoid Apps.
```
Comment (zhuravel) [inline source/android_sdk.rst]:
```suggestion
   :description: Talkable has an Android framework enabling you to use Talkable in Android apps.
```

File: source/campaigns/campaign_placements.rst [hunk 4]
```diff
@@ -1,6 +1,9 @@
 .. _campaigns/campaign_placements:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Campaign Placements let you find the best place for your campaign, so your customers will never miss them.
```
Comment (zhuravel) [inline source/campaigns/campaign_placements.rst]:
```suggestion
   :description: Campaign Placements let you find the best place for your campaign, so your customers will never miss it.
```

File: source/campaigns/campaign_placements/post_purchase.rst [hunk 5]
```diff
@@ -1,6 +1,10 @@
 .. _campaigns/campaign_placements/post_purchase:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: With the Post Purchase placement, your Campaign will show up automatically as a popup once the customer made a successful purchase on your site.
+
+
```
Comment (zhuravel) [inline source/campaigns/campaign_placements/post_purchase.rst]: Remove extra blank line.

File: source/campaigns/editor/videos.rst [hunk 6]
```diff
@@ -1,6 +1,9 @@
 .. _campaigns/editor/videos:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: There are videos for you to dive in Liquid. Learn how you can navigate your liquid customize screens within  Talkable.
```
Comment (zhuravel) [inline source/campaigns/editor/videos.rst]:
```suggestion
   :description: There are videos for you to dive in Liquid. Learn how you can customize screens within Talkable.
```

File: source/campaigns/editor/filters.rst [hunk 7]
```diff
@@ -1,6 +1,9 @@
 .. _campaigns/editor/filters:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Standard liquid filters that are available out of box.
```
Comment (zhuravel) [inline source/campaigns/editor/filters.rst]:
```suggestion
   :description: Standard Liquid filters that are available out of box.
```

File: source/campaigns/tutorials/global.rst [hunk 8]
```diff
@@ -1,6 +1,9 @@
 .. _campaigns/tutorials/global:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Learn how to work with multiple ZeroClipboard instances.
```
Comment (zhuravel) [inline source/campaigns/tutorials/global.rst]: This page covers other topics besides ZeroClipboard.

File: source/campaigns/tutorials/offers_show.rst [hunk 9]
```diff
@@ -1,6 +1,9 @@
 .. _campaigns/tutorials/offers_show:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Instant reward campaign lets you reward your brand advocate immediately — right after a referral link to a friend was sent.
```
Comment (zhuravel) [inline source/campaigns/tutorials/offers_show.rst]: Advocate/Friend sometimes capitalized in `meta` tags but not always.
Please choose one variant (I prefer capitalized) and stick to it everywhere.

File: source/campaigns/views/friend_rewards_paid.rst [hunk 10]
```diff
@@ -1,6 +1,9 @@
 .. _campaigns/views/friend_rewards_paid:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: It's good to give thanks to a friend. Send a reward via email, and remind them about it later on.
```
Comment (zhuravel) [inline source/campaigns/views/friend_rewards_paid.rst]:
```suggestion
   :description: It’s good to give thanks to a friend. Send a reward via email, and remind them about it later on.
```

File: source/api_v2/origins.rst [hunk 11]
```diff
@@ -1,6 +1,9 @@
 .. _api_v2/origins:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: The Talkable Origins API allows the registration of ‘off-site’ or ‘backend’ purchases, or CRM events. That means they can be used in the referral flow.
```
Comment (zhuravel) [inline source/api_v2/origins.rst]:
```suggestion
   :description: The Talkable Origins API allows the registration of “off-site” or “backend” purchases, or CRM events. That means they can be used in the referral flow.
```

File: source/campaigns/views/notifier_offers_share_via_email.rst [hunk 12]
```diff
@@ -1,6 +1,9 @@
 .. _campaigns/views/notifier_offers_share_via_email:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Send an invite email to a friend showing their special share link. Make the email looks personal — just like it's written for them.
```
Comment (zhuravel) [inline source/campaigns/views/notifier_offers_share_via_email.rst]:
```suggestion
   :description: Send an invite email to a Friend showing their special share link. Make the email looks personal — just like it’s written for them. 
```

File: source/index.rst [hunk 13]
```diff
@@ -5,6 +5,9 @@
 
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Talkable campaigns, features and integration docs for referral program launch and settings
```
Comment (zhuravel) [inline source/index.rst]:
```suggestion
   :description: Talkable campaigns, features and integration docs for referral program launch and settings.
```

File: source/integration/custom/overview.rst [hunk 14]
```diff
@@ -1,6 +1,9 @@
 .. _integration/custom/overview:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: You can integrate Talkable referral marketing platform to e-commerce sites a few quick steps.
```
Comment (zhuravel) [inline source/integration/custom/overview.rst]:
```suggestion
   :description: You can integrate Talkable referral marketing platform to e-commerce sites in a few quick steps.
```

File: source/integration/custom/questions.rst [hunk 15]
```diff
@@ -1,6 +1,9 @@
 .. _integration/custom/questions:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: This is a collection of your Frequently asked Questions (FAQs). If your question isn’t in this list, just ask us.
```
Comment (zhuravel) [inline source/integration/custom/questions.rst]:
```suggestion
   :description: This is a collection of your Frequently Asked Questions (FAQs). If your question isn’t in this list, just ask us.
```

File: source/integration/demandware.rst [hunk 16]
```diff
@@ -1,6 +1,9 @@
 .. _integration/demandware:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Talkable cartridge is a self-contained cartridge, which you can easily integrate into any salesforce commerce cloud project.
```
Comment (zhuravel) [inline source/integration/demandware.rst]:
```suggestion
   :description: Talkable cartridge is a self-contained cartridge, which you can easily integrate into any Salesforce Commerce Cloud project.
```

File: source/integration/magento/magento1.rst [hunk 17]
```diff
@@ -1,6 +1,9 @@
 .. _integration/magento/magento1:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: A Magento Talkable extension (direct download) is available for easy integration. The extension supports Magento versions 1.5+
```
Comment (zhuravel) [inline source/integration/magento/magento1.rst]:
```suggestion
   :description: A Magento Talkable extension is available for easy integration. The extension supports Magento versions 1.5+
```

File: source/integration/shopify.rst [hunk 18]
```diff
@@ -1,6 +1,9 @@
 .. _integration/shopify:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Here’s how you can you can integrate Talkable Shopify app with your store.
```
Comment (zhuravel) [inline source/integration/shopify.rst]:
```suggestion
   :description: Here’s how you can integrate Talkable Shopify app with your store.
```

File: source/ios_sdk.rst [hunk 19]
```diff
@@ -1,6 +1,9 @@
 .. _ios_sdk:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Talkable has an iOS framework enabling you to use Talkable in iOS Apps
```
Comment (zhuravel) [inline source/ios_sdk.rst]:
```suggestion
   :description: Talkable has an iOS framework enabling you to use Talkable in iOS apps.
```

File: source/web_hooks/claim_signup.rst [hunk 20]
```diff
@@ -1,6 +1,9 @@
 .. _web_hooks/claim_signup:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: The Talkable Friend Email Gating Webhook notifies your endpoint that friend email gating form was submitted.
```
Comment (zhuravel) [inline source/web_hooks/claim_signup.rst]:
```suggestion
   :description: Friend Email Gating Webhook notifies your endpoint that a Friend email gating form was submitted.
```

File: source/web_hooks/create_coupon.rst [hunk 21]
```diff
@@ -1,6 +1,9 @@
 .. _web_hooks/create_coupon:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Talkable Create Coupon Webhook enables automatic coupon creation. That means you are no longer need to upload coupon lists into the Talkable platform manually.
```
Comment (zhuravel) [inline source/web_hooks/create_coupon.rst]:
```suggestion
   :description: Create Coupon Webhook enables automatic coupon creation. That means you are no longer need to upload coupon lists into the Talkable platform manually.
```

File: source/web_hooks/check_unsubscribe.rst [hunk 22]
```diff
@@ -1,6 +1,9 @@
 .. _web_hooks/check_unsubscribe:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Talkable Check Unsubscribe Webhook ensures that emails are not sent to unsubscribed users.
```
Comment (zhuravel) [inline source/web_hooks/check_unsubscribe.rst]:
```suggestion
   :description: Check Unsubscribe Webhook ensures that emails are not sent to unsubscribed users.
```

File: source/web_hooks/event.rst [hunk 23]
```diff
@@ -1,6 +1,9 @@
 .. _web_hooks/event:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: The Talkable Event Webhook lets you know when a purchase was made.
```
Comment (zhuravel) [inline source/web_hooks/event.rst]:
```suggestion
   :description: Event Webhook lets you know when a purchase was made.
```

File: source/web_hooks/offer_signup.rst [hunk 24]
```diff
@@ -1,6 +1,9 @@
 .. _web_hooks/offer_signup:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: The Talkable Advocate Signup Webhook notifies your endpoint of an advocate signup forms submission.
```
Comment (zhuravel) [inline source/web_hooks/offer_signup.rst]: I believe using "Talkable" or "The Talkable" here is redundant.
```suggestion
   :description: Advocate Signup Webhook notifies your endpoint of an advocate signup forms submission.
```

File: source/web_hooks/post_share.rst [hunk 25]
```diff
@@ -1,6 +1,9 @@
 .. _web_hooks/post_share:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Talkable Post Share Webhook provides notification of share events performed within a referral campaign workflow.
```
Comment (zhuravel) [inline source/web_hooks/post_share.rst]:
```suggestion
   :description: Post Share Webhook provides notification of share events performed within a referral campaign workflow.
```

File: source/web_hooks/referral.rst [hunk 26]
```diff
@@ -1,6 +1,9 @@
 .. _web_hooks/referral:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: The Talkable Referral Webhook notifies your endpoint that your brand advocate’s referral status “Approved” or “Unblocked”.
```
Comment (zhuravel) [inline source/web_hooks/referral.rst]:
```suggestion
   :description: Referral Webhook notifies your endpoint that your brand Advocate’s referral status has become “Approved” or “Unblocked”.
```

File: source/web_hooks/reward.rst [hunk 27]
```diff
@@ -1,6 +1,9 @@
 .. _web_hooks/reward:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: The Talkable Reward Webhook notifies your endpoint that a reward is ready to be given, or it has been already given.
```
Comment (zhuravel) [inline source/web_hooks/reward.rst]:
```suggestion
   :description: Reward Webhook notifies your endpoint that a reward is ready to be given, or it has been already given.
```

File: source/web_hooks/unsubscribe.rst [hunk 28]
```diff
@@ -1,6 +1,9 @@
 .. _web_hooks/unsubscribe:
 .. include:: /partials/common.rst
 
+.. meta::
+   :description: Talkable Unsubscribe Webhook notifies you that a user unsubscribed from Talkable referral email.
```
Comment (zhuravel) [inline source/web_hooks/unsubscribe.rst]:
```suggestion
   :description: Unsubscribe Webhook notifies you that a user has unsubscribed from Talkable referral email.
```

File: source/campaigns/tutorials/global.rst (hunk 8 above)
Comment (zhuravel) [reply source/campaigns/tutorials/global.rst]: @AlexKom9 Please uncover hidden conversations on GitHub and fix them too.

Comment (AlexKom9) [reply source/campaigns/tutorials/global.rst]: yes ofc

Comment (AlexKom9) [reply source/campaigns/tutorials/global.rst]: @zhuravel could you please help to write correct description for this page

Comment (zhuravel) [reply source/campaigns/tutorials/global.rst]: @iurevych Please advise as a reporter of the JIRA task.

Comment (iurevych) [reply source/campaigns/tutorials/global.rst]: @AlexKom9 you can use it as a description: `Learn how to work with multiple ZeroClipboard instances`

**Pull request merged**